qemu/tests/cfg/block_detect_zeroes.cfg

```diff
@@ -0,0 +1,35 @@
+- block_detect_zeroes:
+    type = block_detect_zeroes
+    data_image="stg1"
+    images += " ${data_image}"
+    drv_extra_params_stg1 += "discard=unmap"
+    blk_extra_params_stg1 += ",serial=stg1"
+    image_name_stg1 = images/stg1
+    image_size_stg1 = 2G
+    remove_image_stg1 = yes
+    force_create_image_stg1 = yes
+    Linux:
+        guest_cmd = "dd if=/dev/zero of={0} oflag=direct bs=1M count=1000"
+        pre_guest_cmd = "mkfs.xfs -f {0} && mkdir -p /home/{0} && "
+        pre_guest_cmd += "mount {0} /home/{0}"
+        post_guest_cmd = "mkfs.xfs -f `losetup -f --show /home/{0}/test.dat`"
+        post_guest_cmd += " && losetup -l && losetup -D && umount -fl /home/{0}"
+    Windows:
+        guest_cmd = "fsutil file createnew {0} 1024000000"
+    ovmf:
+        no i440fx
+    variants:
+        - off:
+            drv_extra_params_stg1 += ",detect-zeroes=off"
+        - on:
+            drv_extra_params_stg1 += ",detect-zeroes=on"
+        - unmap:
+            drv_extra_params_stg1 += ",detect-zeroes=unmap"
+    variants:
+        - boot:
```

Comment:
The sub-variant should not use the same name as the top variant, otherwise these cases will be run when `only boot`.

```
 (1/4) io-github-autotest-qemu.block_detect_zeroes.boot.off.arm64-pci: STARTED
 (1/4) io-github-autotest-qemu.block_detect_zeroes.boot.off.arm64-pci: CANCEL: Test cancelled due to --dry-run (0.00 s)
 (2/4) io-github-autotest-qemu.block_detect_zeroes.boot.on.arm64-pci: STARTED
 (2/4) io-github-autotest-qemu.block_detect_zeroes.boot.on.arm64-pci: CANCEL: Test cancelled due to --dry-run (0.00 s)
 (3/4) io-github-autotest-qemu.block_detect_zeroes.boot.unmap.arm64-pci: STARTED
 (3/4) io-github-autotest-qemu.block_detect_zeroes.boot.unmap.arm64-pci: CANCEL: Test cancelled due to --dry-run (0.00 s)
 (4/4) io-github-autotest-qemu.boot.arm64-pci: STARTED
 (4/4) io-github-autotest-qemu.boot.arm64-pci: CANCEL: Test cancelled due to --dry-run (0.00 s)
```

@XueqiangWei @qingwangrh, please help handle this issue asap, thanks.

Reply:
Sorry for this issue, thank you for pointing it out. Qing is handling it.